src/mscorlib/src/System/Reflection/Assembly.cs

```diff
@@ -1268,11 +1239,11 @@ internal bool IsSafeForReflection()
                                              IntPtr pPrivHostBinder,
                                              bool throwOnFileNotFound,
                                              bool forIntrospection,
-                                             bool suppressSecurityChecks, IntPtr ptrLoadContextBinder = default(IntPtr))
+                                             IntPtr ptrLoadContextBinder = default(IntPtr))
         {
             return _nLoad(fileName, codeBase, assemblySecurity, locationHint, ref stackMark,
                 pPrivHostBinder,
-                throwOnFileNotFound, forIntrospection, suppressSecurityChecks, ptrLoadContextBinder);
+                throwOnFileNotFound, forIntrospection, true, ptrLoadContextBinder);
```

Comment:
>true [](start = 55, length = 4)

nit, it is nice to prefix bools 
like /* suppressSecurityChecks = */ true
but I see this is not widely done.